module/VuFind/src/VuFind/Search/Solr/HierarchicalFacetHelper.php

```diff
@@ -179,11 +183,14 @@ public function flattenFacetHierarchy($facetList)
      * @param bool   $allLevels   Whether to display all levels or only
      * the current one
      * @param string $separator   Separator string displayed between levels
+     * @param string $domain      Translation domain for default translations
+     * of a multilevel string or empty string to omit translation
```

Comment:
Maybe I'm misunderstanding, but isn't it possible that we might want translation without a specific text domain? Would it be better to either have separate flags for domain vs. translate/non-translate, or else at least look for an explicit `false` rather than a "false-y" value?

Reply:
Domain is 'default' for translation without a specific text domain. I'll change non-translation to false anyway to clarity.